tests/test_core.py

```diff
@@ -468,7 +468,6 @@ def test_register_flow_with_saturn_does_everything():
 
         flow = TEST_FLOW.copy()
         assert flow.storage is None
-        assert isinstance(flow.environment, LocalEnvironment)
```

Comment:
this was broken by `prefect` 0.14.0, but it's not important to the use of `prefect-saturn`. So it's safe to just remove this test, which probably shouldn't have been here anyway.